### ISSUE TITLE
'View Gizmos' hides selection box

### ISSUE THREAD

**Issue by ryevdokimov** [godotengine/godot#87800]
### Tested versions

v4.2.1.stable.mono.official [b09f793f5]

### System information

Godot v4.2.1.stable.mono - Windows 10.0.22631 - Vulkan (Forward+) - integrated AMD Radeon(TM) Graphics (Advanced Micro Devices, Inc.; 31.0.23013.1023) - AMD Ryzen 7 PRO 6850U with Radeon Graphics (16 Threads)

### Issue description

Title

Is this the intended behavior? 

Should the selection box be part of the group of gizmos it hides? 

### Steps to reproduce

https://github.com/godotengine/godot/assets/105675984/3c030144-cfa5-4dd2-b4ae-a97eb0894e09

### Minimal reproduction project (MRP)

N/A

**Comment by Calinou** [comment]
Yes, this is an intentional change: https://github.com/godotengine/godot/pull/48620

**Comment by ryevdokimov** [comment]
Understood.
